finance/funding_requests.md

```diff
@@ -0,0 +1,81 @@
+
+
+The Astropy project wants to make life easy for all the volunteers and people working towards our shared goal; on the other hand, we also need to make sure we spend grant money in a fiscally responsible way and as allowed by donors or funding agencies. This policy thus aims to strike a balance between as little bureaucracy as possible while also ensuring that money is spend responsibly and that Astropy has all the documentation that we required for any of the grants given to Astropy.
+
+
+
+Funding requests can be made to the Interim Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+If you have questions on how to apply the rules, please email finance@astropy.org .
+
+
+## Travel
+
+Travel requests should be made as early as possible and at least 1 month before travel.   Requests made less than 1 month before the travel is to occur will still be considered but should include a justification for the timing.  
+
+1. Send an email to finance@astropy.org with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+* Advance needed
+
+Costs for per diem can use the [US Government per diem rates | https://www.gsa.gov/travel/plan-book/per-diem-rates].  These should be considered maximum values for the trips and request less than that amount can be submitted for reimbursement.
+
+Flights should be booked in economy class.   If the flight is over 4 hours
+and the additional cost is less than 20%, a higher class of seating may be
+booked.  We encourage travel plans to be optimize their travel for
+productivity.   If additional costs are required, a justification should be 
+submitted for approval.  Medical reasons are an acceptable justification 
+for higher costs.   
+
+In general, contributors should attempt to minimize the environmental impact
+of their travel. 
+
+2. The interim finance committee will aim to provide approval within 1 week of the request.
+
+
+3.  Travel occurs.
+
+4.  Preferable within two weeks of completing the trip, travel costs are submitted via the [NumFOCUS Rocket form | https://numfocus.org/rocket]
+
+5. NumFOCUS provides the information for approval to the Finance Committee.
+
+6. At least, one member of the finance committee provides approval for the payment.
+
+7. Payment is made by NumFocus, typically within 30 days.
+
+
+## Contractor Work
+
+This document describes how the process has been handled in the past, when funding was available through specific grants for very well defined tasks. The interim finance committee and the community as a whole are engaging in a discussion how to handle this going forward.
+
+1.  A request for contract work should be made via email to finance@astropy.org.
+
+2.  If the request is not coming from the coordination committee, the finance committee will seek approval for this contract from the Coordination Committee   
```

Comment:
Allowing for the possibility like we've done thus far of the CoCo saying "this is some contract work that's desired" and just not requiring a second "is that ok?"
```suggestion
2.  If the request is not coming from or already solicited by the coordination committee, the finance committee will seek approval for this contract from the Coordination Committee   
```